File: frontend/lib/models/gift.dart
class Gift {
  final int id;
  final String name;
  final String icon;
  final int price;
  final String type;

  Gift({required this.id, required this.name, required this.icon, required this.price, required this.type});

  factory Gift.fromJson(Map<String, dynamic> json) => Gift(
    id: json['id'] ?? 0,
    name: json['name'] ?? '',
    icon: json['icon'] ?? '',
    price: json['price'] ?? 0,
    type: json['type'] ?? 'NORMAL',
  );
}

File: backend/src/main/java/com/voiceapp/model/Gift.java
package com.voiceapp.model;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;
import java.time.LocalDateTime;

@Data
@TableName("gift")
public class Gift {
    @TableId(type = IdType.AUTO)
    private Long id;
    private String name;
    private String icon;
    private Integer price;
    private String type;
    private Integer sortOrder;
    private LocalDateTime createdAt;
}
